forem/forem
# Fix bug that arises when  a previously tagged comment is deleted

## What type of PR is this? (check all applicable)


## Description
See [Issue #10061](https://github.com/forem/forem/issues/10061). If a user edits their comment that contained a Comment Tag to a comment that no longer exists, upon saving their edits, the save-action fails silently; navigating to the user's original comment also 404's.
This bug fix returns the message `"Comment Not Found"` if a previously-tagged comment no longer exists.

**Additional Considerations:**
We may want to fix the silent failure of the save-action, perhaps display some sort of error message. This is beyond the scope of this PR. 

## QA Instructions, Screenshots, Recordings
1. Fire up your dev environment and sign in using two different profiles (use incognito windows).
2. As Profile1, create a Comment (1) and note its permalink id_code
3. As Profile2, create a Comment (2) and Tag Comment1 in it
4. Delete Comment1
5. Edit Comment2 and save. You should now see the message "Comment Not Found" inside Comment1's liquid tag


## Added tests?


## Added to documentation?


## [optional] Are there any post deployment tasks we need to perform?
No

## [optional] What gif best describes this PR or how it makes you feel?

![Bug Spray](https://media.giphy.com/media/d1TwYpDcR1zPi/giphy.gif)
